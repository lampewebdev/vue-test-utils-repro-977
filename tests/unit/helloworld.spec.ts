import { shallowMount } from "@vue/test-utils";
import HelloWorld from "../../components/HelloWorld.vue";

describe("HelloWorld", () => {
    test("should match snapshot", async () => {
        const wrapper = shallowMount(HelloWorld, {
            propsData: {
                msg: "some msg",
            },
        });
        expect(wrapper.element).toMatchSnapshot();
        wrapper.unmount();
    });
});