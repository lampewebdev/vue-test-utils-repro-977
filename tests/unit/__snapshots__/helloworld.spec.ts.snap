// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld should match snapshot - info 1`] = `
<h1>
  some msg
</h1>
`;

exports[`HelloWorld should match snapshot 1`] = `
<h1>
  some msg
</h1>
`;
